CONTRIBUTING.md

```diff
@@ -67,7 +67,21 @@ The default policy for taking contributions is “Squash and Merge” - because
 
 ### Make sure your patch works for all supported versions of swift
 
-The CI will do this for you.  You can use the docker-compose files included if you wish to check locally.  Currently all versions of swift >= 5.7 are supported.  For example usage of docker compose see the main [README](./README.md#an-alternative-using-docker-compose)
+The CI will do this for you, but a project maintainer must kick it off for you.  Currently all versions of Swift >= 5.8 are supported.
```

Comment:
I wonder if we should just have a single place where supported versions are defined and refer to that.  Or specify as last 3 minor versions here like it is elsewhere to stop us having to update multiple places every 6 months.

Reply:
I wondered that too, but it felt like a more significant rejig to perhaps do in a more structured way next?